src/libtls/tls_crypto.c

```diff
@@ -2553,10 +2554,16 @@ CALLBACK(destroy_key_types, void,
 	ht->destroy_function(ht, (void*)free);
 }
 
-/*
- * See header.
+/**
+ * Create an enumerator over supported key types within a specific TLS range
+ *
+ * Enumerates over key_type_t
+ *
+ * @param min_version	minimum negotiated TLS version
+ * @param max_version	maximum negotiated TLS version
+ * @return				enumerator of key types
```

Comment:
We don't use Doxygen comments for private helper functions.

```suggestion
```
The line above maybe too because the description already explains that key types are enumerated.